Makefile

```diff
@@ -77,9 +78,18 @@ docker_test_lint:
 .PHONY: docker_generate_docs
 docker_generate_docs:
 	docker run --rm -it \
+	 	-e ENABLE_BPMETADATA=1 \
 		-v "$(CURDIR)":/workspace \
 		$(REGISTRY_URL)/${DOCKER_IMAGE_DEVELOPER_TOOLS}:${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} \
-		/bin/bash -c 'source /usr/local/bin/task_helper_functions.sh && generate_docs'
+		/bin/bash -c 'source /usr/local/bin/task_helper_functions.sh && generate_docs display'
+
+# Generate metadata
+.PHONY: docker_generate_metadata_w_display
```

Comment:
Is this function needed?  The `docker_generate_docs` will already call `generate_metadata` once `ENABLE_BPMETADATA` is added above.

Reply:
Will it work for local generation too?

Reply:
Since you are setting it to `1`, I think so, but haven't tested.  Here is the code: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/main/infra/build/developer-tools/build/scripts/task_helper_functions.sh#L322

That said, should be fine to leave this as well, just more a note that the above should also do it.